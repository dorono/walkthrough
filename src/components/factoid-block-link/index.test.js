import React from 'react';
import {shallow} from 'enzyme';
import FactoidBlockLink from './index';

describe('FactoidBlockLink', () => {
    it('should render without errors', () => {
        shallow(<FactoidBlockLink />);
    });

    it('should match snapshot', () => {
        expect(shallow(<FactoidBlockLink />)).toMatchSnapshot();
    });
});
