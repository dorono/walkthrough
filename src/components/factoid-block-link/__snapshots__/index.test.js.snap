// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FactoidBlockLink should match snapshot 1`] = `
<BlockLink
  isLink={true}
  type="fblock"
/>
`;
